### PR TITLE
Review B3 - Krampus

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: packs/data/pathfinder-bestiary-3.db/krampus.json [hunk 1]
```diff
@@ -1801,7 +1773,7 @@
...
...rabbed]{Grabbed}, @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 1}, and subjected to Krampus's regression ability at the start of each round it is captured.</p>\n<p>If the victim @UUID[Compendium.pf2e.actionspf2e.Escape]{Escapes} (DC 48), it climbs out of the basket. A captured creature can attack the interior of the basket, but only with unarmed attacks or weapons of light Bulk or less, against an AC of 40. If the captured creature deals 40 slashing or piercing damage, the basket is broken and all the creatures inside spill out. A creature that gets free by either Escaping or by someone breaking the basket is deposited in a square of their choosing adjacent to Krampus.</p>\n<p>The basket is an extradimensional space, so the weight of the creatures within doesn't count against Krampus's Bulk limit. Up to four Small creatures or two Medium creatures can fit in the basket at one time.</p>\n<p>Krampus can @UUID[Compendium.pf2e.actionspf2e.Repair]{Repair} the basket as a 2-action activity that doesn't require tools. If the basket is lost or destroyed, Krampus can create a new one with 1 hour of work. The basket functions as a normal sack if used by anyone other than Krampus.</p>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-3.db/krampus.json:1776]: Should the Athletics check have `action:shove` as a trait?

Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/krampus.json:1776]: Hm, good question. Not sure if it's actually a shove action. On the other hand, it won't hurt to add it I guess?

Comment (SpartanCPA) [reply packs/data/pathfinder-bestiary-3.db/krampus.json:1776]: It would allow bonus to Fort DC against Shove to proc

Comment (MSAbaddon) [inline packs/data/pathfinder-bestiary-3.db/krampus.json:1776]:
```suggestion
                    "value": "<p><strong>Requirements</strong> Krampus has a Medium or smaller creature @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed}</p>\n<hr />\n<p><strong>Effect</strong> Krampus attempts to stuff the grabbed creature into his basket, attempting an @Check[type:athletics|traits:action:shove] check against the target's Reflex DC. If he succeeds, the creature is shoved into Krampus's basket and Krampus's spiked chain or claw is freed for future Strikes.</p>\n<p>The captured creature is @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed}, @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 1}, and subjected to Krampus's regression ability at the start of each round it is captured.</p>\n<p>If the victim @UUID[Compendium.pf2e.actionspf2e.Escape]{Escapes} (DC 48), it climbs out of the basket. A captured creature can attack the interior of the basket, but only with unarmed attacks or weapons of light Bulk or less, against an AC of 40. If the captured creature deals 40 slashing or piercing damage, the basket is broken and all the creatures inside spill out. A creature that gets free by either Escaping or by someone breaking the basket is deposited in a square of their choosing adjacent to Krampus.</p>\n<p>The basket is an extradimensional space, so the weight of the creatures within doesn't count against Krampus's Bulk limit. Up to four Small creatures or two Medium creatures can fit in the basket at one time.</p>\n<p>Krampus can @UUID[Compendium.pf2e.actionspf2e.Repair]{Repair} the basket as a 2-action activity that doesn't require tools. If the basket is lost or destroyed, Krampus can create a new one with 1 hour of work. The basket functions as a normal sack if used by anyone other than Krampus.</p>"
```

Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/krampus.json]: Added the trait

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**